test/foods-test.js

```diff
@@ -0,0 +1,71 @@
+var assert    = require('chai').assert;
+var webdriver = require('selenium-webdriver');
+var until = webdriver.until;
+var test      = require('selenium-webdriver/testing');
+var frontEndLocation = "http://localhost:8080"
+var Food = require('../lib/food')
+var pry = require('pryjs')
+var $ = require('jquery');
+var host = require('../lib/config').host
+
+test.describe('testing food index', function() {
+  var driver;
+  this.timeout(1000000);
+
+  test.beforeEach(function() {
+    driver = new webdriver.Builder()
+      .forBrowser('chrome')
+      .build();
+  })
+
+  test.afterEach(function() {
+    driver.quit();
+  })
+
+  test.it("lists all the foods on load", function() {
+    // var foodCount = Food.getAllFoods().then(function(data){
```

Comment:
@case-eee turns out it is a bigger issue with being able to make AJAX calls in our test - we had a session on it with Mike this morning, and it looks like we will continue down the path we are on for now, though it is not ideal. not sure if @lao9 has any more thoughts!